proposals/generic-ref-struct.md

```diff
@@ -0,0 +1,424 @@
+Unconstraining our generics
+=====
+
+## Summary
+This proposal is an aggregation of several different proposals for making `ref struct` more generally usable in our type system. The goal being to unify code paths today that have to specialize on `ref struct` and other types do to the inability of `ref struct` to participate in generics and implement interfaces. This proposal will allow us to generalize on `ref struct` as naturally as other types. 
```

Comment:
Well, I was going to try and read through this without correcting spelling and grammar, but ya got me good Jared.

```suggestion
This proposal is an aggregation of several different proposals for making `ref struct` more generally usable in our type system. The goal is to unify code paths today that have to specialize on `ref struct` and other types due to the inability of `ref struct` to participate in generics and implement interfaces. This proposal will allow us to generalize on `ref struct` as naturally as other types. 
```